src/main/webapp/app/entities/course/courseUtil.ts

```diff
@@ -0,0 +1,25 @@
+import { ICourse } from 'app/shared/model/course.model';
+import { defaultDateTime } from 'app/shared/util/date-utils';
+import { v4 as uuidv4 } from 'uuid';
+
+export const initCourse: () => ICourse = () => {
+  const currentDate = new Date();
+  const oneYearFromNow = new Date();
+  oneYearFromNow.setFullYear(currentDate.getFullYear() + 1);
+
+  return {
+    code: '',
+    name: '',
+    ownerId: 0, // Provide a default ownerId
+    description: null,
+    invitationCode: uuidv4(), //TODO: add real invitation code
```

Comment:
I think it should be a pattern like `code` -`courseId`. Example: code: 22CLC1, courseId: 2. invitationCode = 22CLC1-2. Because we can give this invitation code to user for joining to our class, so the uuidv4 is too long

Reply:
I just add a random invitation code, we will handle later